contrib/seekable_format/zstdseek_decompress.c

```diff
@@ -252,6 +252,7 @@ size_t ZSTD_seekable_free(ZSTD_seekable* zs)
 
 ZSTD_seekTable* ZSTD_seekTable_create_fromSeekable(const ZSTD_seekable* zs)
 {
+    if (zs->seekTable.entries == NULL) return NULL;
```

Comment:
Since we are dereferencing `zs` here, I would prefer an `assert(zs != NULL);` just before this statement.

Reply:
Thanks for your review. I added the assert